plugin/src/main/java/com/redhat/jenkins/nodesharingfrontend/SharedNode.java

```diff
@@ -45,6 +47,8 @@
         @Override public String getShortDescription() { return "ReservationTasks should not run here"; }
     };
 
+    private boolean wipeout = true;
```

Comment:
The filed initializer here will be used only when instance is created via constructor which it typically is not as we use xstream deserialization to instantiate. This should be set by `readResolve()` or the semantics will needs to be inverted so the default value, `false`, would achieve what we want.